endpoints/api/tag.py

```diff
@@ -416,3 +421,62 @@ def post(self, namespace, repository, tag):
             repo_name=repository,
         )
         return "", 200
+
+
+@resource("/v1/repository/<apirepopath:repository>/tag/<tag>/pull_statistics")
+@path_param("repository", "The full path of the repository. e.g. namespace/name")
+@path_param("tag", "The name of the tag")
+class RepositoryTagPullStatistics(RepositoryParamResource):
+    """
+    Resource for retrieving pull statistics for a specific repository tag.
+    """
+
+    @require_repo_read(allow_for_superuser=True, allow_for_global_readonly_superuser=True)
+    @disallow_for_app_repositories
+    @nickname("getTagPullStatistics")
+    def get(self, namespace, repository, tag):
+        """
+        Get pull statistics for a specific tag.
+        """
+        if not features.IMAGE_PULL_STATS:
```

Comment:
this will work, but we do have a `@show_if(features.IMAGE_PULL_STATS)` decorator as well